static/src/js/components/ProjectPanels/__tests__/VariableTreePanel.test.js

```diff
@@ -89,6 +89,32 @@ describe('VariableTreePanel', () => {
     ])
   })
 
+  test('displays a message when there are no items to display', () => {
+    const { enzymeWrapper } = setup({
+      accessMethods: {
+        opendap: {
+          hierarchyMappings: [],
+          keywordMappings: [],
+          selectedVariables: [],
+          variables: {
+            'V123456-EDSC': {
+              meta: {},
+              umm: {}
+            },
+            'V987654-EDSC': {
+              meta: {},
+              umm: {}
+            }
+          }
+        }
+      }
+    })
+
+    expect(enzymeWrapper.exists()).toBeTruthy()
```

Comment:
Is this assertion needed if we have the one below it?

Reply:
Ah I forgot to remove this one. I was using that line for debugging just showing/hiding the element.